pkgs/development/tools/rust/sqlx-cli/default.nix

```diff
@@ -17,7 +17,8 @@ rustPlatform.buildRustPackage rec {
   cargoBuildFlags = [ "-p sqlx-cli" ];
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ openssl ];
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks;
```

Comment:
Is openssl required on darwin or only on linux?

Reply:
Nice catch! I don't know how you had the intuition, but you were right.